tests/test_notify.py

```diff
@@ -168,15 +168,16 @@ def test_notify_user_filter(self):
         self.assertEqual('Man', event['user']['lastname'])
 
     def test_notify_payload_matching_filter(self):
-        bugsnag.configure(params_filters=['number'])
-        bugsnag.notify(ScaryException('unexpected failover'),
-                       apple='four', number=76)
+        bugsnag.configure(params_filters=['bar'])
+        bugsnag.notify(ScaryException('unexpected failover'), foo=4, bar=76)
```

Comment:
This change is to pull all of the `bugsnag.notify` call onto one line, which gets around a small difference in how Python 3.8 reports line numbers in stack traces — see b4f3095664456572af1b83da691bded0f13222fb and https://bugs.python.org/issue12458 for more information

I think this is a bit nicer than doing a dynamic version check to see if the line number should be 172 or 173